engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -65,6 +66,31 @@ namespace dmGameSystem
         SpriteResourceOverrides() : m_Material(0) {}
     };
 
+    const uint32_t MAX_TEXTURE_COUNT = dmRender::RenderObject::MAX_TEXTURE_COUNT;
+    const uint8_t CACHE_EVICTION_FRAMES = 10; // how many frames cache entry is stored before eviction
+
+    struct AnimationData
+    {
+        dmDoubleLinkedList::ListNode    node;
+        // Used after resolving info from all textures
+        dmhash_t                        m_AnimationID;                      // The animation of the driving atlas
+        uint32_t                        m_Frames[MAX_TEXTURE_COUNT];        // The resolved frame indices
+        uint32_t                        m_LastAccessTick;
+        uint32_t                        m_CacheKey;
```

Comment:
Added cache key as a field because in case when cache entry will be removed because of cache eviction we should know the key to correctly update hashtable.